test/test_datapipe.py

```diff
@@ -2332,15 +2332,15 @@ def unbatch(x):
 class TestSerialization(TestCase):
     @skipIfNoDill
     def test_spawn_lambdas_iter(self):
-        idp = dp.iter.IterableWrapper(range(3)).map(lambda x: x + 1)
+        idp = dp.iter.IterableWrapper(range(3)).map(lambda x: x + 1).shuffle()
```

Comment:
The lack of `shuffle` raises warning since `shuffle=True` later for `DataLoader`.